client/search-ui/src/components/LastSyncedIcon.tsx

```diff
@@ -14,15 +14,34 @@ interface Props {
 }
 
 export const LastSyncedIcon: React.FunctionComponent<React.PropsWithChildren<Props>> = props => {
-    const formattedTime = format(Date.parse(props.lastSyncedTime), 'yyyy-MM-dd pp')
+    const parsedDate = new Date(Date.parse(props.lastSyncedTime))
+    const formattedTime = format(parsedDate, 'yyyy-MM-dd pp')
+    const oneDayAgo = new Date()
+    const oneWeekAgo = new Date()
+    oneDayAgo.setDate(oneDayAgo.getDate() - 1)
+    oneWeekAgo.setDate(oneWeekAgo.getDate() - 7)
+
+    let color = 'currentColor'
+    let status = `Last synced: ${formattedTime}`
+    let icon = mdiCloudCheckOutline
+    if (parsedDate < oneWeekAgo) {
+        color = 'var(--danger)'
+        status = 'Warning: severely out of date, last synced: ' + formattedTime + '. Please contact your administrator.'
```

Comment:
string templates ftw :)

```suggestion
        status = `Warning: severely out of date, last synced: ${formattedTime}. Please contact your administrator.`
```

Reply:
What if the user _is_ the administrator? Can we change the message then and say they should check the repo settings page and see whether it's still being synced/mirrored?

Reply:
@mrnugget I like that, we would need to pass the user all the way down here but I think it makes sense.

Reply:
After talking with @mrnugget and @limitedmage we decided that it might be a bit more trouble than its worth passing the authenticatedUser object down to here due to the large dependency tree.